### DISCUSSION TITLE
Install Issue macOS with Proxy

### DISCUSSION THREAD

**Discussion by dlashua** [johnlindquist/kit#1196] [Errors]
An earlier version of ScriptKit installed just fine for me. But some time in the not to distant past, updates failed to work.

I stashed away my scripts, deleted all the relevant directories and tried a fresh install. It would hang up on the step trying to install Node. I think it's having issues with our corporate proxy. I have the relevant environment variables set (http_proxy, https_proxy) but I think perhaps it's not honoring them. 

I followed the instructions at #1056 and that seemed to get me up and running though, on first run, it did reinstall the SDK and ENV. That seemed to find the proxy just fine, so I think perhaps it's just the Node install step that has issues.

Happy to give more details if needed. Point me in the right direction of where the Node Install happens, and I can likely fix it myself and submit a PR.

**Comment by johnlindquist** [comment]
@dlashua Can you show me:

~/Library/Logs/Kit/main.log

The install process logs to that file and might have some clues as to what's going on.
